public/css/signup.css

```diff
@@ -0,0 +1,74 @@
+
+h1{
+    text-align: center;
+    font-family: Roboto;
+
+}
+
+.msg{
+    text-align: center;
+    font-family: Roboto;
+    color: red;
+    
+}
+.containerr , .submitdiv{
+    /* display: grid;
+    grid-template-columns: repeat( auto-fill, minmax(400px, 1fr) );
+    grid-gap:10px; */
+    margin:0 auto;
+    width:70%;
+}
+.c{
+    display: inline-block;
+    width: 45%;
+    margin:0 20px 0 0;
+
+}
+
+
+.c>.item>input[type=text], [type=email],[type=password],[type=tel],textarea {
```

Comment:
I don't understand how this selector works
please tell me why you put `.c>.item>input` only before the first one